src/index.js

```diff
@@ -63,6 +64,7 @@ export default (WrappedComponent) => {
     const getPlayerStatus = () => getStateForPlayer('currentStatus');
 
     useImperativeHandle(ref, () => ({
+      ...videoRef.current,
       mux: { emit }
     }));
```

Comment:
Could you please change this to use [`Object.assign`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)? Instead of creating a new object, we should assign the `mux` property to the existing `video` element.
```suggestion
    useImperativeHandle(ref, () => Object.assign(videoRef.current, {
      mux: { emit }
    }));
```